dev/com.ibm.ws.microprofile.faulttolerance_git_fat_tck/build.gradle

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+task clone {
+  mustRunAfter cleanFat
+  gitClone "microprofile-fault-tolerance"
```

Comment:
The `gitClone` line needs to be in a `doLast` block, otherwise you'll do the clone at configuration time rather than at task execution time.